boot.py

```diff
@@ -190,27 +195,39 @@ def on_post_window_command(self, window: sublime.Window, command_name: str, args
                 sublime.set_timeout(panel_manager.update_log_panel)
 
 
-class LspOpenLocationCommand(sublime_plugin.TextCommand):
+class LspOpenLocationCommand(LspWindowCommand):
```

Comment:
I found two places that ran this command from the view:
 - https://github.com/sublimelsp/LSP-volar/blob/32f0c42ecf2d10036a913518938a91d704c809f1/plugin.py#L69-L69
 - https://github.com/sublimelsp/LSP-typescript/blob/e6ae40de6c3322fbe064698ff49f958ab6b3e4f9/commands.py#L35-L35